app/controllers/course_summaries_controller.rb

```diff
@@ -1,6 +1,9 @@
 class CourseSummariesController < ApplicationController
   include CourseSummariesHelper
 
+  before_filter      :authorize_only_for_admin,
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.